client/undo-manager.js

```diff
@@ -59,8 +60,9 @@ class UndoManager {
       return false;
 
     // do some comparisons here to change for changes
-    console.log(serial)
-    console.log(this.current)
+    console.log("serial", serial) // updated
+    utils.storage.backup(serial);
+    console.log("current", this.current)
```

Comment:
Are these logs important, or are they just for debugging?

Reply:
They are for debugging, but they have not to be dropped manually, considering that there are much more of them in a code base. And actually I need them in dev environment. So please accept unchanged, I provide solution for your concern in next PR, closing issue #358.